Removed js files and explicit naming of dependencies for minifaction

rajansen opened the pull request:
Might not be complete  yet...

ShMcK commented:
What are these requests intended to do?
It seems most of the commits add, then immediately erase what the change with the next commit.

In Git there are a few stages:

add: a staging are for adding something until you've completed a goal
commit: the final record of all goals completed

If possible, try to limit the number of commits to changes you want in the final product.

rajansen commented:
Hi Shawn,
I removed all js files and added the meteor typescript pacakage since it wouldnt use the .ts files at all. And i added the names of dependencies because when minified in production they would fail ( the names can no longer be inferred from the function arguments )
Sorry for the messy commits, i'm a git beginner...and tried some stuf to get it to run on heroku ( still not completely succesfull, but in seems to run in production locally now)
grruud

ShMcK commented:
That's really cool. I'm happy you chose this project as your first pull request.
